src/main/java/pl/gacik/tictac/TurnHolder.java

```diff
@@ -0,0 +1,25 @@
+package pl.gacik.tictac;
+
+import java.util.Queue;
+import java.util.concurrent.LinkedBlockingQueue;
+
+public class TurnHolder {
+
+    private Queue<Player> turnQueue = new LinkedBlockingQueue<>();
+
+    public TurnHolder(Player... players) {
+        for (Player player :players) {
+            turnQueue.add(player);
+        }
+    }
+
+    /**
+     * @return Player whose the next turn is
+     */
+    public Player getNext() {
```

Comment:
getNextPlayer() maybe?
It is defined in TurnHolder, so getNext() suggests that it will return next Turn or TurnHolder.

Reply:
But I really like mechanism :)